pkg/apis/workflow/v1alpha1/container_set_template_types_test.go

```diff
@@ -5,8 +5,18 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/yaml"
 )
 
+func validateContainerSetTemplate(yamlStr string) error {
+	var tmpl Template
+	err := yaml.Unmarshal([]byte(yamlStr), &tmpl)
```

Comment:
I keep needing a `MustUnmarshal(interface{})` function - let me create a PR to add it

Reply:
Great! I can rebase after #5785 to make use of that. Thanks!